packages/legend-graph/src/graphManager/action/changeDetection/DSLService_ObserverHelper.ts

```diff
@@ -98,7 +185,9 @@ export const observe_MultiExecutionTest = skipObserved(
   },
 );
 
-export const observe_ServiceTest = (metamodel: ServiceTest): ServiceTest => {
+export const observe_ServiceTest_Legacy = (
```

Comment:
though we rename these models to `DEPRECATED__...`, I think we can keep calling these methods `observe_LegacyServiceTest` for now. What do you think?